### PR TITLE
Optiktv

### PR DESCRIPTION
**Pull request by jas-lamba**

### CODE REVIEW

File: mytelus-analytics/input/actions.json [hunk 1]
```diff
@@ -142,5 +142,39 @@
...
+                "actionName":"optiktv_plan_card_click",
+                "aa.event.name" : "optiktv_plan_card_click", 
+                "aa.event.action" : "internal_campaign_click",
+                "aa.event.detail" : "card_campaign",
+                "aa.internal.promo.click.event" : "1",
+                "aa.internal.promo.list" : "optiktv_plan_card",
+                "aa.internal.promo.name" : "optiktv_plan_card",
+                "&&events": "event150"
+            },
+            "exit_link":{
```
Comment (prateem) [inline mytelus-analytics/input/actions.json]: exit_link_click

Comment (jas-lamba) [reply mytelus-analytics/input/actions.json]: made the change in latest commit.

File: mytelus-analytics/input/actions.json [hunk 2]
```diff
@@ -142,5 +142,39 @@
...
+                "aa.internal.promo.click.event" : "1",
+                "aa.internal.promo.list" : "optiktv_plan_card",
+                "aa.internal.promo.name" : "optiktv_plan_card",
+                "&&events": "event150"
+            },
+            "exit_link_click":{
+                "include": "page_views:plans.optik.subscription",
+                "change_theme":{
+                    "actionName":"exit_optik_change_theme",
+                    "aa.exit.link.event" : "1",
```
Comment (prateem) [inline mytelus-analytics/input/actions.json]: common attribute should be extracted to the parent

**Comment by prateem** [comment]
Actually while you need to push a commit, do you think the `change_theme` exit link event should be pluralized to `change_themes` or even fully actualized to `change_theme_packs`?

**Comment by jas-lamba** [comment]
`change_theme_packs` sounds better

File: mytelus-analytics/input/actions.json (hunk 2 above)
Comment (jas-lamba) [reply mytelus-analytics/input/actions.json]: made the change in latest commit.

File: mytelus-analytics/input/actions.json [hunk 3]
```diff
@@ -159,19 +159,17 @@
             },
             "exit_link_click":{
                 "include": "page_views:plans.optik.subscription",
-                "change_theme":{
+                "include":"aa.exit.link.event:1",
```
Comment (prateem) [inline mytelus-analytics/input/actions.json]: this should just be `"aa.exit.link.event": 1`, not `"include": "aa.exit.link.event:1"`

Comment (prateem) [reply mytelus-analytics/input/actions.json]: the include directive is to include entire other payloads from a file

Comment (jas-lamba) [reply mytelus-analytics/input/actions.json]: updated

**Pull request merged**